build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -341,7 +341,9 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         [Target]
         public static BuildTargetResult BuildProjectsForNuGetPackages(BuildTargetContext c)
         {
-            if (CurrentPlatform.IsWindows)
+            string rid = c.BuildContext.Get<string>("TargetRID");
+
+            if (CurrentPlatform.IsWindows || rid == "ubuntu.14.04-arm" || rid == "ubuntu.16.04-arm" || rid == "tizen.4.0.0-armel" )
```

Comment:
What about Debian armel?

Also, @schellap @eerhardt Do you know why is this built only for Windows?